src/axom/primal/geometry/NURBSCurve.hpp

```diff
@@ -777,17 +882,46 @@ class NURBSCurve
    * \param [in] t parameter value at which to evaluate
    * \param [out] n1 First output NURBS curve
    * \param [out] n2 Second output NURBS curve
-   * \param [in] normalize Whether to normalize the output curves
+   * \param [in] normalizeParameters Whether to normalize the output curves
    * 
    * If t is at the knot u_0 or u_max, will return the original curve and
    *  a degenerate curve with the first or last control point
    *
    * \pre Requires \a t in the *strict interior* of the span of the knots
+   * 
+   * \return True iff the curve was split (i.e., t is in the knot span)
```

Comment:
"iff" typo

Reply:
That's supposed to be the math shorthand for "if and only if," but it' probably unnecessary. I'll change it here and elsewhere.

Reply:
Ok, I've never seen anyone use that.

Reply:
It is actually pretty common in math literature. But programmers might get confused, so I would spell it out.